packages/desktop-client/src/components/modals/CreateLocalAccountModal.tsx

```diff
@@ -75,15 +86,17 @@ export function CreateLocalAccountModal({
                     const name = event.target.value.trim();
                     setName(name);
                     if (name && nameError) {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏since we now have multiple types of errors: I think we can reset to the "no error" state if there previously was an error instead of using the `name` var
```suggestion
                    if (nameError.error) {
```